mithril-stm/src/schnorr_signature/mod.rs

```diff
@@ -0,0 +1,173 @@
+// TODO: Remove
+#![allow(dead_code)]
+
+mod signature;
+mod signing_key;
+pub(super) mod utils;
+mod verification_key;
+
+pub use signature::*;
+pub use signing_key::*;
+pub use utils::*;
+pub use verification_key::*;
+
+use dusk_jubjub::Fq as JubjubBase;
+
+/// A DST (Domain Separation Tag) to distinguish between use of Poseidon hash
+pub(crate) const DST_SIGNATURE: JubjubBase = JubjubBase::from_raw([0u64, 0, 0, 0]);
```

Comment:
I guess that this is internal stuff of the `schnorr_signature` module:
```suggestion
const DST_SIGNATURE: JubjubBase = JubjubBase::from_raw([0u64, 0, 0, 0]);
```